doozerlib/cli/images_streams.py

```diff
@@ -608,7 +609,27 @@ def images_upstreampulls(runtime):
     print(yaml.dump(retdata, default_flow_style=False, width=10000))
 
 
-def reconcile_jira_issues(runtime, pr_links: Dict[str, str], dry_run: bool):
+def connect_issue_with_pr(pr: PullRequest.PullRequest, issue: str):
+    """
+    Aligns an existing Jira issue with a PR. Put the issue number in the title of the github pr.
+    Args:
+        pr: The PR to align with
+        issue: JIRA issue key
+    """
+    if issue in pr.title:  # the issue already in pr title
+        return
+    elif "OCPBUGS" in pr.title:  # another issue is in pr title, add comment
```

Comment:
Let's be more concise. Instead of testing OCPBUGS in PR title, check if `matches = re.findall(r'OCPBUGS-[0-9]+', pr.title)` is empty, where you did in line 625.

Reply:
updated